tests/_project/project/src/Pyz/Glue/AuthRestApi/Object.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
After the test be finished this file become tracked by git as updated file. Need to put in gitignore and created it in the beginning of the test and remove after